conf/defaults.ini

```diff
@@ -318,6 +318,8 @@ key_file =
 skip_verify = false
 from_address = admin@grafana.localhost
 from_name = Grafana
+# EHLO identity in SMTP dialog (defaults to instance_name)
```

Comment:
Could you please move this comment from `default.ini` to `sample.ini`. This file should contain as few comments as possible.

Reply:
Looks good otherwise. This can be merged once the config comments has been moved.